### ISSUE TITLE
Can't `pip install`

### ISSUE THREAD

**Issue by dimitrovskif** [VapourApps/va_master#1]
The `libvirt-python` dependency causes the following error during `pip install`.

```bash
 Running setup.py install for libvirt-python ... error
    Complete output from command /usr/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-Dx0e5x/libvirt-python/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-9JBpWf-record/install-record.txt --single-version-externally-managed --compile:
    Package libvirt was not found in the pkg-config search path.
    Perhaps you should add the directory containing `libvirt.pc'
    to the PKG_CONFIG_PATH environment variable
    No package 'libvirt' found
    Package libvirt was not found in the pkg-config search path.
    Perhaps you should add the directory containing `libvirt.pc'
    to the PKG_CONFIG_PATH environment variable
    No package 'libvirt' found
    running install
    running build
    /usr/bin/pkg-config --print-errors --atleast-version=0.9.11 libvirt
    Package libvirt was not found in the pkg-config search path.
    Perhaps you should add the directory containing `libvirt.pc'
    to the PKG_CONFIG_PATH environment variable
    No package 'libvirt' found
    error: command '/usr/bin/pkg-config' failed with exit status 1
```

* OS: Linux Mint 64bit 18
* Version: 0.0.11 (1e18e08)

**Comment by dimitrovskif** [comment]
Should be a part of #22

**Comment by dimitrovskif** [comment]
Fixed in #3b848499928d8c44cb80178518c7797d1bca1115

**Comment by dimitrovskif** [comment]
Fixed in 3b84849
